feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.html

```diff
@@ -4,6 +4,7 @@
       <button
         class="cx-btn btn btn-block btn-primary cx-add-to-cart-btn"
         (click)="onAddToCart(container.configuration, container.routerData)"
+        (keyup)="onAddToCart(container.configuration, container.routerData)"
```

Comment:
The event is fired after reaching the `Add to Cart / Continue to Cart` button via `Tab` key without pressing `Enter` or `Space` keys

Reply:
This is definitely not the same behaviour as on the product detail page.

Reply:
The question is what behaviour is expected!?!

Reply:
Just excluded the line 7, since it is not necessary.